src/Sql/Sql/ChangeLog.md

```diff
@@ -19,6 +19,7 @@
 -->
 ## Upcoming Release
 * Add support for SQL DB Hyperscale tier
+* Removed unnecessary parameters passed in different restore mode
```

Comment:
This makes it sound like the powershell command line parameters were changed, which sounds a bit scary. Can you rephrase to state that it was a bug fix? e.g. "Fixed bug where restore could fail due to setting unnecessary properties in restore request"

Reply:
updated.